configs/dyhead/atss_r50_fpn_dyhead_for_reproduction_1x_coco.py

```diff
@@ -0,0 +1,111 @@
+_base_ = [
```

Comment:
The name for_reproduction might be confusing. Can we keep the naming convention and rename it as atss_r50_caffe_fpn_dyhead_1x_coco.py?